tests/apple-exporter/components/test-class-video.php

```diff
@@ -73,6 +73,39 @@ public function testFilter() {
 		);
 	}
 
+	/**
+	 * Tests the ability for the Video component to get and save caption information
+	 *
+	 * @access public
+	 */
+	public function testCaption() {
+		$workspace = $this->prophet->prophesize( '\Exporter\Workspace' );
+
+		// Pass the mock workspace as a dependency
+		$component = new Video( '<figure class="wp-block-video"><video controls="" src="http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ElephantsDream.mp4"/><figcaption>caption</figcaption></figure>',
+			$workspace->reveal(), $this->settings, $this->styles, $this->layouts );
+
+		// Test.
+		$this->assertEquals(
+			array(
+				'role' => 'container',
+				'components' => array(
+					array(
+						"role" => "video",
+						"URL" => "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ElephantsDream.mp4",
```

Comment:
Same comment here about using a mock Video component similar to the mock Audio component to avoid the external request. We should also change these URLs to something that is clearly fake.